src/vivarium_gates_iv_iron/constants/data_values.py

```diff
@@ -54,6 +54,16 @@ class _HemoglobinDistributionParameters(NamedTuple):
         for row in _hemoglobin_threshold_data[key]:
             _htd.append((state, "Female", *row))
 
-HEMOGLOBIN_THRESHOLD_DATA = pd.DataFrame(_htd,
-                                         columns=["pregnancy_status", "sex", "age_start", "age_end", "severe",
-                                                  "moderate", "mild"])
+HEMOGLOBIN_THRESHOLD_DATA = pd.DataFrame(
+    _htd,
+    columns=["pregnancy_status", "sex", "age_start", "age_end", "severe", "moderate", "mild"]
+)
+
+
+class _MaternalBMI(NamedTuple):
+    ANEMIA_THRESHOLD = 10.0
```

Comment:
I think this will need to be 100 rather than 10 - sorry I had some inconsistent unit definitions. Hemoglobin exposure is in g/L (threshold of 100), but I referred to it in a couple of places as 10 g/dL